src/frontend/src/eth/utils/token.utils.ts

```diff
@@ -10,3 +10,6 @@ export const isSupportedErc20TwinTokenId = (tokenId: TokenId): boolean =>
 
 export const isNotSupportedErc20TwinTokenId = (tokenId: TokenId): boolean =>
 	!isSupportedErc20TwinTokenId(tokenId);
+
+export const tokenAddressToHex = (address: string): string =>
```

Comment:
- For `0x` we can use `mapAddressStartsWith0x`

- For converting to hex:
   - Isn't there already an EtherJS or existing ic-js function we can reuse?
   - If none, can you provide some tests.

Reply:
for sure, we can use `hexZeroPad` from ethers (doc: https://docs.ethers.org/v4/api-utils.html#hex-strings); it does everything

changed